### PR TITLE
PortAuthority rolled out to HA

### PR DESCRIPTION
**Pull request by lassewesth**
PortAuthority moves to neo4j-com. HA integration tests can run in parallel.

### CODE REVIEW

File: enterprise/ha/src/test/java/org/neo4j/test/ha/ClusterIT.java [hunk 1]
```diff
@@ -152,30 +172,44 @@ public void testInstancesWithConflictingClusterPorts() throws Throwable
...
+    /**
+     * This test has been green because second database failed to start over directory conflict,
+     * not port conflict.
+     *
+     * With the directory conflict out of the way we start up but presumably are in a bad state.
+     *
+     * However, detecting bad state seems impossible here.
+     */
     @Test
+    @Ignore( "The failure mode for HA port conflict is what?" )
```
Comment (davidegrohmann) [inline enterprise/ha/src/test/java/org/neo4j/test/ha/ClusterIT.java:184]: delete the test if it doesn't make sense

Comment (MishaDemianenko) [inline enterprise/ha/src/test/java/org/neo4j/test/ha/ClusterIT.java:184]: please do not ignore test, if test useless or wrong update it or remove

**Review by MishaDemianenko — COMMENTED** [review]
One small bit of ignorance detected! otherwise it just great! fantastic! 🎉

**Comment by lassewesth** [comment]
I'll take the issue of the commented-out test up with @digitalstain directly

**Pull request merged**